### ISSUE TITLE
Can’t craft item

### ISSUE THREAD

**Issue by JQE** [Imprex-Development/zero-inventory-problems#19]
### Description

I add the materials in the crafting table. It shows on the right but I can’t remove it. 

### Reproduction Steps

I add the materials in the crafting table. It shows on the right but I can’t remove it. 

### Expected Behaviour

I should be able to remove the item. 

### Server Details

Paper 1.20.4

### Zero Inventory Problems Version

1.0.7

**Comment by NgLoader** [comment]
For me is everything working fine on paper 1.20.4

Your using any other plugins with custom recipes?
Are there any console errors?
Have you changed the config file?

**Comment by JQE** [comment]
Yes I am now using a different backpack mod.

No errors in the console. Item jumps to inventory then disappears. Won’t
craft.


Ryan Workman

**Comment by NgLoader** [comment]
Yeah, that's right, when you press shift while crafting, you just need to click on the item without shift.
I will lookout for a fix for that.

**Comment by kn1gh7h4wk** [comment]
I too have this problem, without pressing shift. As OP, I can craft, but cannot put in my inventory. If I press shift, it briefly appears in the inventory and disappears back to the crafting. 
![2024-03-21_10-45-11](https://github.com/Imprex-Development/zero-inventory-problems/assets/49842470/dfce35b0-38ea-477e-b322-49466dd82b8a)

I also noticed that if I deop myself, I cannot craft it at all. Which permission do players need to have to craft it?
![2024-03-21_10-37-59](https://github.com/Imprex-Development/zero-inventory-problems/assets/49842470/36d25c2f-975b-4b90-88db-31569902b47e)

Thanks for your help!

**Comment by NgLoader** [comment]
This issue shout be fixed with the latest release [1.0.8](https://github.com/Imprex-Development/zero-inventory-problems/releases/tag/1.0.8).

Also you can now add permissions for who can craft which item by adding the following setting ``craftingPermission``.
Example:
```yml

  big:
    uniqueName: 'big'
    displayName: '&7Big &eBackpack'
    inventoryRows: 6
    customModelData: 3
    texture: 'ewogICJ0aW1lc3RhbXAiIDogMTYzMzg2MjQwNTg4NSwKICAicHJvZmlsZUlkIiA6ICI1NjY3NWIyMjMyZjA0ZWUwODkxNzllOWM5MjA2Y2ZlOCIsCiAgInByb2ZpbGVOYW1lIiA6ICJUaGVJbmRyYSIsCiAgInNpZ25hdHVyZVJlcXVpcmVkIiA6IHRydWUsCiAgInRleHR1cmVzIiA6IHsKICAgICJTS0lOIiA6IHsKICAgICAgInVybCIgOiAiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9kYTMzMTY5YjcyY2Y4OTE4YjgyYzViZDI3Y2JhOWVlNmMwZWI4OTE2NDA0MDQ4MGJiMjdmODUzNGUwZmE1ODA3IiwKICAgICAgIm1ldGFkYXRhIiA6IHsKICAgICAgICAibW9kZWwiIDogInNsaW0iCiAgICAgIH0KICAgIH0KICB9Cn0='
    lore:
      - '&aA Big Backpack'
    recipe:
      discover: true
      group: 'null'
      patternOne: 'LCL'
      patternTwo: 'ECE'
      patternThree: 'DCD'
      patternMapping:
        L: LEATHER
        E: LEAD
        C: CHEST
        D: DIAMOND
    craftingPermission: 'zeroinventoryproblems.crafting.big'
```

When no permission is provided or the permission is empty everyone can craft the item.

**Comment by NgLoader** [comment]
This issue shout be fixed with the latest release [1.0.8](https://github.com/Imprex-Development/zero-inventory-problems/releases/tag/1.0.8).
